sql/core/src/test/scala/org/apache/spark/sql/streaming/FileStreamSourceSuite.scala

```diff
@@ -1153,174 +1153,213 @@ class FileStreamSourceSuite extends FileStreamSourceTest {
     }
   }
 
-  test("max files per trigger") {
-    withTempDir { case src =>
-      var lastFileModTime: Option[Long] = None
+  test("maxFilesPerTrigger & maxBytesPerTrigger threshold logic must be obeyed") {
```

Comment:
Please don't touch the existing old test case. Since we don't allow both threshold. The should be independently tested. The code duplication can be refactored into a test helper method instead of making a big single test case.

Also, please use the test prefix like the following. 
```scala
- test("maxFilesPerTrigger & maxBytesPerTrigger threshold logic must be obeyed") {
+ test("SPARK-46641: maxBytesPerTrigger threshold logic must be obeyed") {
```

Reply:
Should I use prefix `SPARK-46641` for all `maxBytesPerTrigger` related tests or only this one?

Reply:
fixed